Making Sense/Models/CarCRUD.cs

```diff
@@ -25,15 +26,7 @@ private void Save()
         public Car Get(int id)
         {
             List<Car> Cars = JsonConvert.DeserializeObject<List<Car>>(Read());
-            Car aux = new Car();
-            foreach (var car in Cars)
-            {
-                if(car.CarID == id)
-                {
-                    aux = car;
-                }
-            }
-            return aux;
+            return Cars.Single(car => car.CarID == id);
```

Comment:
Muy bueno! Encontré [este](https://www.dotnettricks.com/learn/linq/understanding-single-singleordefault-first-and-firstordefault) articulo con las diferencias entre `Single`, `SingleOrDefault`, `First` y `FirstOrDefault`. Creo que `Single` es la que mas aplica en este caso

Reply:
Excelente! Lo voy a leer, gracias!